tests/integration.rs

```diff
@@ -251,3 +251,35 @@ fn test_call() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn test_response() -> Result<()> {
+    let port: u16 = 6489;
+    let _guards = vec![start_redis_server_with_module("response", port)
+        .with_context(|| "failed to start redis server")?];
+    let mut con =
+        get_redis_connection(port).with_context(|| "failed to connect to redis server")?;
+
+    redis::cmd("hset")
+        .arg(&["k", "a", "b", "c", "d", "e", "b", "f", "g"])
+        .query(&mut con)
+        .with_context(|| "failed to run string.set")?;
+
+    let mut res: Vec<String> = redis::cmd("map.mget")
+        .arg(&["k", "a", "c", "e"])
+        .query(&mut con)
+        .with_context(|| "failed to run string.set")?;
+
+    res.sort();
+    assert_eq!(&res, &["a", "b", "b", "c", "d", "e"]);
```

Comment:
I was wondering how this test pass but then I notice that its not, also this client does not really check the reply is resp3 right?